### PR TITLE
Improving efficency of RAKE

### PR DESCRIPTION
**Pull request by sw5678**
Minor changes too the method to make it more efficient from the PR comments.

### CODE REVIEW

**Comment by a3957273** [comment]
Hey @sw5678 , could you resolve the trailing space errors?  I realise they weren't introduced in your commit.

File: src/core/operations/RAKE.mjs [hunk 1]
```diff
@@ -101,22 +101,17 @@ class RAKE extends Operation {
...
-        const wordDegreeMatrix = Array.from(Array(tokens.length), _ => Array(tokens.length).fill(0));
-        phrases.forEach(function (phrase) {
-            phrase.forEach(function (word1) {
-                phrase.forEach(function (word2) {
-                    wordDegreeMatrix[tokens.indexOf(word1)][tokens.indexOf(word2)]++;
-                });
-            });
-        });
+        const wordDegreeMatrix = Array(tokens.length).fill().map(() => Array(tokens.length).fill(0));
+        for (let p=0; p < phrases.length; p++) {
```
Comment (a3957273) [inline src/core/operations/RAKE.mjs:108]: You can improve the readability of this with:

```
for (let phrase of phrases) {
  for (let word1 of phrase) {
    for (let word2 of phrase) {
      wordDegreeMatrix[tokens.indexOf(word1)][tokens.indexOf(word2)]++;
    }
  }
}
```

**Review by a3957273 — CHANGES_REQUESTED** [review]

File: src/core/operations/RAKE.mjs (hunk 1 above)
Comment (sw5678) [reply src/core/operations/RAKE.mjs:108]: Sorry have only just seen this now. Ill make the changes.

**Comment by a3957273** [comment]
Very nice improvements!

**Pull request merged**